### ISSUE TITLE
Lots of repeated text in the result

### ISSUE THREAD

**Issue by patdelphi** [Const-me/Whisper#118]
Hi,

When I tried to transcribe the Japanese video, always got a lot of repeated text in the result asn attached sceen snap, is it possible to fix this bug?
![Uploading 微信截图_20230611121202.png…]()

**Comment by hongjiapeng** [comment]
Which model are you using?

**Comment by 2233431914** [comment]
I have the same question,I am using ggml-large.bin model, GPU RTX 3060Ti.Language is Chinese,the record is 16 minutes long. The front of the result is fine.But  since about the 7 minute output became the "oh" in Chinese.

**Comment by darkelf123** [comment]
yes,I have the same question。The voice content of the last few minutes of the video is missing，and output a lots of "oh".

**Comment by cascadahk** [comment]
same issue here, hope that will be fixed in the next update

**Comment by emcodem** [comment]
Did you look at #26

**Comment by zlbbme** [comment]
same problem，Once the same words and sentences appear consecutively, they will continue to repeat themselves

**Comment by emcodem** [comment]
Solution is easy: just don't put the repeated text and tokens that force hallucinations into the prompt. As the program does currently not do that in an automatgically way, your only workaround is to use -mc 0 parameter which might reduce output quality but definitely prevents forever loops.
